server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileDao.java

```diff
@@ -34,7 +34,6 @@
 import org.sonar.db.DatabaseUtils;
```

Comment:
The commit message is too long. I suggest: `SONAR-8889 api/qualityprofiles/delete also cleans up changelog and index`